pixeldata/Cargo.toml

```diff
@@ -26,3 +26,7 @@ num-traits = "0.2.12"
 [dev-dependencies]
 dicom-test-files = "0.2.0"
 rstest = "0.6.4"
```

Comment:
I recall that you introduced this dependency. I noticed that the latest version is at 0.10. How hard would it be to update it?